lib/services/display.js

```diff
@@ -395,6 +400,26 @@ module.exports = class DisplayService extends Schmervice.Service {
     return registeredUsers[0].users;
   };
 
+  filterClassDetails = async (singleClass, sequence) => {
+    const exerciseDetails = {};
+    const contentEx = [];
+    const newExercise = {};
+    exerciseDetails.id = singleClass.id;
+    exerciseDetails.title = singleClass.title;
+    exerciseDetails.description = singleClass.description;
+    exerciseDetails.course_id = singleClass.course_id;
+    exerciseDetails.exercise_id = singleClass.exercise_id;
+    exerciseDetails.lang = singleClass.lang;
+    exerciseDetails.type = singleClass.type;
+    exerciseDetails.meet_link = singleClass.meet_link;
+    exerciseDetails.facilitator = singleClass.facilitator;
```

Comment:
If you're using Lodash, can you use [pick](https://lodash.com/docs/4.17.15#pick) for this?
```suggestion
    const exerciseDetails = _.pick(singleClass, ['id', 'title', 'description',
        'course_id', 'exercise_id', 'lang', 'type', 'meet_link', 'facilitator']);
    const contentEx = [];
    const newExercise = {};
```
One potential difference with `pick` will be that if `singleClass` does not have a given property, then `Object.keys(exerciseDetails)` won't either.  So for example, with your code, if `singleClass` didn't have a `lang` property, then `exerciseDetails.lang` would be `undefined` but `lang` would still appear in `Object.keys(exerciseDetails)`.  It would **not** if you use `pick`.  But it will still appear if `singleClass.lang` is `undefined` or `null`.